runtime.sh

```diff
@@ -1,675 +0,0 @@
-__ALLOC=1 # Starting heap at 1 because 0 is the null pointer.
```

Comment:
This file is out of date and can be regenerated with `gcc make-sh-runtime.c -o make-sh-runtime.exe && ./make-sh-runtime.exe`.